Cookbook/Proposals/SplitOctopus.md

```diff
@@ -0,0 +1,40 @@
+# Split Octopus codebase
+
+* Author(s): Giorgos Tsiapaliokas
+* Review Managers: David Rodrigues, Yasuhiro Inami, Olivier Halligon
+
+## Introduction
+
+From the early versions of the codebase Octopus was the place were features were added.
+Unfortunately Octopus hasn't adapted as the rest of the codebase has.
+In the iOS codebase we use frameworks in order to organize the code and Octopus isn't one.
+
+## Motivation
+
+In it's current state Octopus is collection of services and features which is consumed
+by each target this requires some manual work for each target.
+
+## Proposed solution
+
+We propose to convert Octopus into a framework with the name `BabylonOctopus`.
```

Comment:
Should we not call it `BabylonAppServices` instead of `BabylonOctopus`?

The original motivation for the name, because it has no back bone, is funny but not very relevant anymore.